src/less/workingfiles/submit.less

```diff
@@ -250,17 +251,21 @@ td {
     background-color: @super-light-gray;
 }
 
+
 #queue-table th:first-child {
-    width: 22%;
+
 }
 
+div .table-cell-div{
+    overflow: scroll;
```

Comment:
Yeah I centred a lot as well so the page should have dynamic proportions rather than specific margin percentages etc.

O.k cool I'll take out the scroll bar for as many browsers as possible and remove the bold font weight with the h4 tag.